presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -135,6 +135,7 @@
 
```

Comment:
typos in commit message: 
 - to temporarlIy iNGnore table
 - `provider will loNg the corruption`